src/client/app/translations/data.ts

```diff
@@ -27,7 +27,7 @@ const LocaleTranslationData = {
 		"AreaUnitType.none": "no unit",
 		"ascending": "Ascending",
 		"as.meter.unit": "as meter unit",
-		"as.meter.defaultgraphicunit": "as meter unit",
+		"as.meter.defaultgraphicunit": "as meter unit",//Ask
```

Comment:
I noticed the message is the same as the line above it in all languages, so I was unsure if "as.meter.defaultgraphicunit" should be deleted in favor of the generalized use of "as.meter.unit" or if the message associated with as.meter.defaultgraphicunit should be "as meter default graphic unit" instead of "as meter unit"

Reply:
I see and good catch. The as.meter.defaultgraphicunit should be "as meter default graphic unit". Could you update all the languages?